### PR TITLE
adding generic ontology queries

### PR DESCRIPTION
**Pull request by zhlu9890**

### CODE REVIEW

File: stored_queries/generic_ontology/generic_ontology_get_ancestors.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Get all ancestors (all parent's parents) for this term
```
Comment (jayrbolton) [inline stored_queries/generic_ontology/generic_ontology_get_ancestors.yaml]: @zhlu9890 just to make this a little less long to read and write, can you rename the files from `generic_ontology` to just `ontology`?
For example: `generic_ontology/generic_ontology_get_ancestors.yaml` -> `ontology/ontology_get_ancestors.yaml`

**Comment by jayrbolton** [comment]
I had one comment above but otherwise it looks good. Will give one more check in a few mins and then merge and release

File: stored_queries/generic_ontology/generic_ontology_get_ancestors.yaml (hunk 1 above)
Comment (scanon) [reply stored_queries/generic_ontology/generic_ontology_get_ancestors.yaml]: I agree with @jayrbolton.  I would rename these all and drop the generic.

**Review by scanon — CHANGES_REQUESTED** [review]
Please rename the directory and queries to drop the "generic_" part.

**Comment by zhlu9890** [comment]
updated name as requested

**Pull request merged**